src/ocsp.c

```diff
@@ -928,6 +946,87 @@ int wolfSSL_i2d_OCSP_REQUEST_bio(WOLFSSL_BIO* out,
     return WOLFSSL_FAILURE;
 }
 #endif /* !NO_BIO */
+
+int wolfSSL_i2d_OCSP_CERTID(WOLFSSL_OCSP_CERTID* id, unsigned char** data)
+{
+    if (id == NULL || data == NULL)
+        return WOLFSSL_FAILURE;
+
+    if (*data != NULL) {
```

Comment:
The behavior of this function differs depending on if *data is null. See https://www.openssl.org/docs/man1.1.0/man3/i2d_OCSP_CERTID.html.